nntrainer/layers/embedding.h

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jijoong Moon <jijoong.moon@samsung.com>
+ *
+ * @file	embedding.h
+ * @date	04 March 2021
+ * @brief	This is Embedding Layer Class of Neural Network
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ */
+
+#ifndef __EMBEDDING_H__
+#define __EMBEDDING_H__
+#ifdef __cplusplus
+
+#include <layer_internal.h>
+#include <tensor.h>
+
+namespace nntrainer {
+
+/**
+ * @class   EmbeddingLayer
+ * @brief   EmbeddingLayer
+ */
+class EmbeddingLayer : public Layer {
```

Comment:
This is not related to this PR, but if this layer's implementation is close to FC layer, you consider extending `FullyConnectedLayer` than `Layer` class in the future PR.

Reply:
I thonght that too.. but I guess It is easier starting from layer.

Reply:
Yes, extend to FCLayer is for later.